CONTRIBUTING.md

```diff
@@ -70,15 +70,22 @@ This creates separate folders in `napari/resources/icons` with full colored icon
 
 Make sure that the file `napari/resources/icons/res.qrc` has a list of all the icons with the correct prefix for each of our themes, e.g. `<file>icons/light/select.svg</file>`, `<file>icons/dark/select.svg</file>`, etc.
 
-Finally, compile all the icons for use in the app by calling.
+Finally, compile all the icons for use in the app by calling either
+
+```sh
```

Comment:
Suggest adding comment to the example calls to show which is pyqt5 and which is pyside2 for clarity.

Reply:
will in a follow-up PR